github/open_close_pr.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3
+
+import github3
+import argparse
+import logging
+import yaml
+from cachecontrol.adapter import CacheControlAdapter
+
+
+REF_FORMAT = 'refs/heads/'
+DEFAULT_COMMIT_MSG = 'automated commit'
+FREEIPA_PRCI_CONFIG_FILE = '.freeipa-pr-ci.yaml'
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+consoleHandler = logging.StreamHandler()
+consoleHandler.setLevel(logging.INFO)
+logger.addHandler(consoleHandler)
+
+
+def load_yaml(yml_path):
+    try:
+        with open(yml_path) as yml_file:
+            return yaml.load(yml_file)
+    except IOError as exc:
+        raise argparse.ArgumentTypeError(
+            'Failed to open {}: {}'.format(yml_path, exc))
+    except yaml.YAMLError as exc:
+        raise argparse.ArgumentTypeError(
+            'Failed to parse YAML from {}: {}'.format(yml_path, exc))
+
+
+class AutomatedPR(object):
+
+    def __init__(self, github_token, repo):
+        github = github3.login(token=github_token)
+        github.session.mount('https://api.github.com',
+                             CacheControlAdapter())
+
+        self.repo = github.repository(repo['owner'], repo['name'])
+
+    def create_branch(self, identifier):
+        last_commit = next(self.repo.commits())
```

Comment:
Will this work if branch is something other than `master`? Or rather, if the latest commit is a in different branch than `args.branch`?